src/operator-manual/metrics/02-reference.md

```diff
@@ -0,0 +1,34 @@
+# Metrics Reference
+
+Kubewarden exposes some relevant metrics that enhance visibility of the platform, and allows cluster
+administrators and policy developers to identify patterns and potential issues.
+
+## Policy Server
+
+The Policy Server is the component that initializes and runs policies. Upon receiving requests from
+the Kubernetes API server, it will forward the request to the policy, and return the response
+provided by the policy to the Kubernetes API server.
+
+### Metrics
+
+> **Note**: Baggage are key-value attributes added to the metric. It is used to enrich the metric
+> with additional information.
```

Comment:
I think it should be plural: `They can be used to enrich...`. That's because it refers to the "key-value attributes"